plugins/lookup/laps.py

```diff
@@ -0,0 +1,233 @@
+# Copyright: (c) 2024, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+name: laps
+author: Nico Ohnezat (@no-12)
+short_description: Inventory plugin for Active Directory
+version_added: 2.2.0
```

Comment:
The next version is going to be `1.5.0`.

```suggestion
version_added: 1.5.0
```